framework/doc/content/source/meshgenerators/ComputeNodePosition.md

```diff
@@ -0,0 +1,17 @@
+# ComputeNodePosition
```

Comment:
what about Parsed(Node)TransformGenerator?

ComputeNode makes me think you re not transforming but generating new values directly

Also Transform lumps it in with out other transform

Reply:
Love it